### PR TITLE
sr v1 checklist

### PR DESCRIPTION
**Pull request by Evgen-Krasnenkov**

### CODE REVIEW

File: src/main/java/mate/academy/service/impl/MovieSessionServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    @Override
+    public MovieSession get(Long id) {
+
+        return movieSessionDao.get(id).orElseThrow(() ->
+                new RuntimeException("Can't get a movie session by id: " + id));
+    }
+
+    @Override
+    public List<MovieSession> findAvailableSessions(Long movieId, LocalDate date) {
+
```
Comment (Sinelife) [inline src/main/java/mate/academy/service/impl/MovieSessionServiceImpl.java:30]: delete empty line

File: src/main/java/mate/academy/service/impl/MovieSessionServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    private MovieSessionDao movieSessionDao;
+
+    @Override
+    public MovieSession add(MovieSession movieSession) {
+        return movieSessionDao.add(movieSession);
+    }
+
+    @Override
+    public MovieSession get(Long id) {
+
```
Comment (Sinelife) [inline src/main/java/mate/academy/service/impl/MovieSessionServiceImpl.java:23]: delete empty line

File: src/main/resources/hibernate.cfg.xml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+        <property name="connection.username">root</property>
+        <property name="connection.password">bvf5u8</property>
+        <property name="show_sql">true</property>
+        <property name="hbm2ddl.auto">update</property>
+
+        <mapping class="mate.academy.model.Movie"/>
+        <mapping class="mate.academy.model.CinemaHall"/>
+        <mapping class="mate.academy.model.MovieSession"/>
+    </session-factory>
+</hibernate-configuration>
```
Comment (Sinelife) [inline src/main/resources/hibernate.cfg.xml:15]: add empty line

Comment (Evgen-Krasnenkov) [reply src/main/resources/hibernate.cfg.xml:15]: done all

**Review by Sinelife — APPROVED** [review]

**Pull request closed without merging**